diffraction/WISH/bragg-detect/cnn/WISHDataSets.py

```diff
@@ -0,0 +1,33 @@
+import torch as tc
+import numpy as np
+import albumentations as A
+from albumentations.pytorch.transforms import ToTensorV2
+from bragg_utils import make_3d_array
+from mantid.simpleapi import Load, Rebunch
+
+
+class WISHWorkspaceDataSet(tc.utils.data.Dataset):
+    def __init__(self, ws_name):
+        ws = Load(Filename = ws_name, OutputWorkspace=ws_name)
```

Comment:
Can we take an existing workspace (by name or by object/handle) rather than load again. WISH will load in the workspace, sometimes do normalisation or additional cropping depending on sample. We should just assert/check that the units are TOF.